src/com/hms_networks/americas/sc/string/StringUtils.java

```diff
@@ -45,4 +48,35 @@ public static String replace(String string, String target, String replacement) {
     // Return modified string
     return stringBuffer.toString();
   }
+
+  /**
+   * Method to split the supplied string into smaller strings using the supplied regex as the
+   * delimiter.
+   *
+   * @param string string to split
+   * @param regex string split delimiter
+   * @return list of string split parts
+   * @since 1.2
+   */
+  public static List split(String string, String regex) {
```

Comment:
Since you are adding the equivalent of
`split(String regex)`
but you might want to just add 
`split(String regex, int limit)`
as well in this PR even though you dont need it right now.

Reply:
I think this is something that should be added as an issue for later since it isn't a simple adaptation of the split() implementation just added.